duckduckgo/iOS
# Manage Duck Addresses

Task/Issue URL: https://app.asana.com/0/1142021229838617/1205057443326659/f

**Description**:
Allows you to manage your private Duck addresses



**Steps to test this PR**:
1. Enable Autofill
2. Create a login that includes a valid private Duck Address
3. Observe the management UI for the address is available
4. Enable and disable the duck Address
5. Observe state changes properly
6. Update the username of the login so the private address is no longer valid
7. Observe the management UI is not available anymore

**Other tests**
- Smoke test email and password generation

**Demo:**
![demo-2](https://github.com/duckduckgo/iOS/assets/1156669/1bd94524-e780-4cbf-9d13-082abc7c2070)


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
